lib/main/pico-sdk/rp2_common/cmsis/stub/CMSIS/Core/Include/m-profile/armv8m_mpu.h

```diff
@@ -188,7 +188,7 @@ typedef struct {
   \brief  Read MPU Type Register
   \return Number of MPU regions
 */
-__STATIC_INLINE uint32_t ARM_MPU_TYPE()
+__STATIC_INLINE uint32_t ARM_MPU_TYPE(void)
```

Comment:
is this change from upstream?
Fixing SDK locally will cause problem later

Reply:
I'll request the change upstream... we've a few bug fixes in F4 libs that make for interesting lib updates

Reply:
https://github.com/raspberrypi/pico-sdk/pull/2095